examples/config-gc.json

```diff
@@ -3,10 +3,47 @@
     "storage": {
         "rootDirectory": "/tmp/zot",
         "gc": true,
-        "gcReferrers": true,
         "gcDelay": "2h",
         "untaggedImageRetentionDelay": "4h",
```

Comment:
Why is this outside the "retention" section?

Reply:
Also make a **new** example file called config-retention.json.

gc != retention